test/cypress/integration/user/dashboard_spec.js

```diff
@@ -1,14 +1,22 @@
 describe('Dashboard', () => {
+
+  const ssDefaultUser = require('../../../fixtures/config/self_service_user.json')
+
   beforeEach(() => {
     cy.setCookie('session', Cypress.env('encryptedSessionCookie'))
     cy.setCookie('gateway_account', Cypress.env('encryptedGatewayAccountCookie'))
   })
 
   describe('Homepage', () => {
+
+    // Use a known configuration used to generate contracts/stubs.
+    // This is also used to generate the session/gateway_account cookies
+    const ssUser = ssDefaultUser.config.users.filter(fil => fil.isPrimary === 'true')[0]
+
     // Note : these from/to datetime strings exactly match those in the pact/contract, so are essential to match against stubs
     // Either change everything together, or map these do a single place like a .json document so the contracts/tests refer to one place
-    const from = encodeURIComponent('2018-05-14T00:00:00+01:00')
-    const to = encodeURIComponent('2018-05-15T00:00:00+01:00')
+    const from = encodeURIComponent(ssUser.sections.dashboard.transaction_summary.from_date)
```

Comment:
not a big fan of unnecessary contractions like ss for selfservice. would a bit of finding and replacing be too much to ask, unless there's a good reason for it